### ISSUE TITLE
const data = ... fetch ... gets too large

### ISSUE THREAD

**Issue by sbeh** [sbeh/iobroker#1]
`const data = ... fetch ... will` end up in a data structure like this:
```
const data = [
  {url: 'https://....', inputs: [1,2,3], labels: [5,6,9]},
  {url: 'https://....', inputs: [2,3,5], labels: [6,9,3]},
  {url: 'https://....', inputs: [3,5,6], labels: [9,,3,2]},
  {...},
  {...}
]
```

The values of `data[...].inputs` and `data[...].labels` come from a time series where a sensor measures a value every n th seconds which gets stored in an sql database.

As you can see, there is a lot of copied data transfferd, because `data[2].inputs[0] = data[1].inputs[1] = data[0].inputs[2]` and the same for `data[...].labels`. This copying was done on the sql server side. This grows the te be transffered data unneded.

I was very helpful the receive the data like this, because the tensors for tensorflow model training could be created very easily:
`const inputs = tf.tensor(data.map(d => d.inputs))
const labels = tf.tensor(data.map(d => d.labels))
model.fit(inputs, labels, ...)`

Too shrink the data transfer, want to receive the data like it is stored in the database:
```
const data = [
  {url: 'https://....', value: 1},
  {url: 'https://....', value: 2},
  {url: 'https://....', value: 3},
  {url: 'https://....', value: 5},
  {url: 'https://....', value: 6},
  {url: 'https://....', value: 9},
  {url: 'https://....', value: 3},
  {url: 'https://....', value: 2},
  {..}
]
```

Change the tensor creation code accordingly so that i can continue using them as parameters for `model.fit(...)`
